tox.ini

```diff
@@ -40,13 +40,6 @@ commands =
 description = Lint Markdown documents following the rules set forth here: https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md
 changedir = {toxinidir}
 allowlist_externals =
-    npm
     npx
-commands_pre = 
-    npm install npm
-    npm i markdownlint-cli
 commands =
-    npx markdownlint-cli README.md CONTRIBUTING.md docs
-commands_post =
-    python -c 'from shutil import rmtree; rmtree("node_modules", True)'
-    python -c 'from os import remove; remove("package-lock.json"); remove("package.json")'
+    npx markdownlint-cli README.md CONTRIBUTING.md docs -i docs/node_modules
```

Comment:
why is `-i` needed? does it try to lint `node_modules`?

Reply:
-i for ignore. It tries to lint READMEs inside docs/node_modules after runing npm install in docs

Reply:
heh yes i meant to ask "does it try .... if you don't pass `-i`". got it thanks